gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -669,16 +685,23 @@ gst_tensor_query_client_chain (GstPad * pad,
   nns_edge_data_set_info (data_h, "client_id", val);
   g_free (val);
 
-  if (NNS_EDGE_ERROR_NONE != nns_edge_send (self->edge_h, data_h)) {
-    nns_logi ("Failed to publish to server node.");
-    goto done;
+  if (self->requested_num > self->max_request) {
```

Comment:
condition to drop the input: self->max_request should be positive value.
if (self->max_request > 0 && self->requested_num > self->max_request)

Reply:
Right. Updated.